src/utils/components/ExportModal/constants.ts

```diff
@@ -82,4 +92,4 @@ export const ALREADY_CREATED_ERROR_CODE = 409;
 
 export const UPSTREAM_UPLOADER_IMAGE = 'quay.io/kubevirt/tekton-tasks:v0.23.0';
 export const DOWNSTREAM_UPLOADER_IMAGE =
-  'registry-proxy.engineering.redhat.com/rh-osbs/container-native-virtualization-kubevirt-tekton-tasks-create-datavolume-rhel9:v4.99.0-50';
+  'registry-proxy.engineering.redhat.com/rh-osbs/kubevirt-tekton-tasks-create-datavolume-rhel9-container-v4.18.0-16';
```

Comment:
Can we start with `registry.redhat.io/container-native-virtualization/kubevirt-tekton-tasks-create-datavolume-rhel9:v4.17` ? We can change to v4.18 before the 4.18 release, but we do not have to.

Reply:
@upalatucci can you send a fix for this?